relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -303,6 +310,28 @@ fn sanitized_span_description(
     Some(sanitized)
 }
 
+/// Trims the given string with the given maximum bytes. Splitting only happens
+/// on char boundaries.
+///
+/// If the string is short, it remains unchanged. If it's long, this method
+/// truncates it to the maximum allowed size and sets the last character to
+/// `*`.
+fn truncate_string(mut string: String, max_bytes: usize) -> String {
+    if string.len() < max_bytes {
```

Comment:
Still think this should be 

```suggestion
    if string.len() <= max_bytes {
```

That is what "max" means, and it would also be consistent with https://github.com/getsentry/relay/blob/0638c8c403e4178dc302791932d067a738aa3bd2/relay-metrics/src/aggregation.rs#L1629